tests/console/wicked_before_test.pm

```diff
@@ -13,13 +13,13 @@
 use base "consoletest";
 use strict;
 use testapi;
-use utils qw(systemctl setup_static_network);
+use utils;
 use mm_network;
 
 sub run {
     my ($self) = @_;
     select_console('root-console');
-    assert_script_run "rcSuSEfirewall2 stop";
+    assert_script_run "rcSuSEfirewall2 stop" unless is_sle() && sle_version_at_least('15');
```

Comment:
Then, shouldn't you stop firewalld ?

Reply:
I have a proof run which shows that test is working without this https://openqa.suse.de/tests/1265210. Give me good reason why I should do this ?